jonathanmusila/Questioner-v1
# #163150002 Refactored code

## What does this PR do?
- Refactors the code and removes the code smells

## Description of the task to be completed
 - [x] Refactor code to remove smells

## How can this be manually tested?
- Go to the repo and navigate to develop branch
- Scroll down to see Readme.md `Maintainability Badge`

### What are the relevant PT stories
#163150002
